pkg/minikube/node/start.go

```diff
@@ -223,6 +229,14 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 	return kcs, config.Write(viper.GetString(config.ProfileName), starter.Cfg)
 }
 
+func validateRuntimeVersion(cr cruntime.Manager) error {
```

Comment:
lets move this helper to cruntime package

so it will be
cruntime.ValidateVersion(cr) and then call cr.Version inside there
there shouldn't be crutnime logic code in node package

Reply:
we already have such function - `cruntime.CompatibleWithVersion()` all cruntime specific logic is there. `validateRuntimeVersion` exists just to make the calling `Start(..)` method a bit more compact

Reply:
the problem is this validate runtime should be done by cruntime package, seems like we are passing func to func back to cruntime....let's do all of that in crunitime package, so we dont have do pass variables and also maintain code in "Node" package not related

Reply:
What exactly is the problem here? What do you mean by `we are passing func to func`?

Reply:
the logic for runtimes needs to be in cruntime package and the the helper func you wrote has nothing that is related to "Node" Package.

so you are passing `cr cruntime.Manager` to node package
and then again from there you again call crutime package

instead of this, it is better the Whole logic be done in cruntime package, 


this code is Easier to write but it is harder to maintain, as someone who needs to maintain this code, please follow the  minikube packages and the reason they were meant to be